assets/src/blocks/amp-story/edit.js

```diff
@@ -48,36 +45,6 @@ class EditPage extends Component {
 		this.onSelectMedia = this.onSelectMedia.bind( this );
 	}
 
-	maybeAddBlockNavigation() {
-		// If no blocks are selected or if it's the current page, change the view.
-		if ( this.props.showBlockNavigation ) {
```

Comment:
Do we still need the determining the value of `showBlockNavigation` in the `withSelect` below or is that redundant now?

Reply:
Indeed unused and can be removed.